modules/@apostrophecms/asset/index.js

```diff
@@ -494,7 +494,13 @@ module.exports = {
           isTask: !argv['check-apos-build'],
           hmr: self.hasHMR(),
           hmrPort: self.options.hmrPort,
-          sourcemaps: self.options.productionSourceMaps
+          sourcemaps: self.options.productionSourceMaps,
+          postcssViewportToContainerToggle: {
```

Comment:
Please document the new options to the dev comments above - what every prop does mean and represents.

Reply:
Right, added a line for `postcssViewportToContainerToggle`